lib/LLVMIRCodeGen/BundleSaver.cpp

```diff
@@ -190,14 +159,16 @@ static void serializeBinaryToText(llvm::StringRef binFileName,
 }
 
 BundleSaver::BundleSaver(const IRFunction *F, const LLVMBackend &llvmBackend)
-    : irgen_(llvmBackend.createIRGen(F, allocationsInfo_)) {
+    : irgen_(llvmBackend.createIRGen(F, allocationsInfo_)),
+      bundleAPI_(BundleApiType::Static) {
```

Comment:
According to the BundleApiType documentation, the default is dynamic. can you please either update the documentation comment or change the default construction to dynamic?

Reply:
Updated the documentation.